chainer/functions/loss/negative_sampling.py

```diff
@@ -170,17 +170,17 @@ def negative_sampling(x, t, W, sampler, sample_size):
 
     .. math::
 
-       f(x, p) = \\log \\sigma(x^\\top w_p) + \\
+       f(x, p) = \\log \\sigma(x^\\top w_p) +  \\\\
        k E_{i \\sim P(i)}[\\log \\sigma(- x^\\top w_i)],
 
-    where :math:`\sigma(\cdot)` is a sigmoid function, :math:`w_i` is the
+    where :math:`\\sigma(\\cdot)` is a sigmoid function, :math:`w_i` is the
     weight vector for the word :math:`i`, and :math:`p` is a positive example.
     It is approximeted with :math:`k` examples :math:`N` sampled from
     probability :math:`P(i)`, like this:
 
     .. math::
 
-       f(x, p) \\approx \\log \\sigma(x^\\top w_p) + \\
+       f(x, p) \\approx \\log \\sigma(x^\\top w_p) +  \\\\
```

Comment:
Does a new line requires? Four backslashes makes two backslashes in TeX that represents a newline in TeX.